File: Examples/Diagram/WinForm/VB.NET/Common/NDiagramExamplesConfig.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Reflection

Imports Nevron.UI
Imports Nevron.Examples.Framework.WinForm

Imports Nevron.Diagram.WinForm.Commands

Namespace Nevron.Examples.Diagram.WinForm
	''' <summary>
	''' Summary description for NDiagramExamplesConfig.
	''' </summary>
	Public Class NDiagramExamplesConfig
		Inherits NExamplesConfig
		#Region "Constructor"

		Public Sub New()
		End Sub


		#End Region

		#Region "Overrides"

		Public Overrides Sub InitDefault()
			MyBase.InitDefault()

			m_SplashImage = NResourceHelper.BitmapFromResource(GetType(NMainForm), "DiagramSplash.png", "Nevron.Examples.Diagram.WinForm.Resources")

			m_sExamplesNamespace = "Nevron.Examples.Diagram.WinForm"
			m_sTreeResource = "ExamplesTree.xml"
			m_sTreeResourcePath = "Nevron.Examples.Diagram.WinForm.Resources"
			m_sFormText = "Nevron Diagram for Windows Forms - Part of Nevron .NET Vision - Examples"

			m_FormIcon = NResourceHelper.IconFromResource(GetType(NMainForm), "Diagram.ico", "Nevron.Examples.Diagram.WinForm.Resources")

			m_sFeedbackString = "mailto:support@nevron.com?subject=Diagram Feedback"

			m_iExampleTreeNodeImageIndex = 11
			m_iExampleTreeNodeSelectedImageIndex = 12

			m_EmbeddedResourcesAssembly = Me.GetType().Assembly
			m_LayoutStrategy = New NDiagramExamplesLayoutStrategy()

			ProductLogo = NResourceHelper.BitmapFromResource(GetType(NMainForm), "DiagramLogo.png", "Nevron.Examples.Diagram.WinForm.Resources")
			ProductName = "Nevron Diagram for .NET"
			ProductAssemblies = New System.Reflection.Assembly(){GetType(NDiagramCommandBarsManager).Assembly}
		End Sub


		#End Region
	End Class
End Namespace

File: Examples/Diagram/WinForm/CSharp/Common/NDiagramExamplesConfig.cs
using System;
using System.Reflection;

using Nevron.UI;
using Nevron.Examples.Framework.WinForm;

using Nevron.Diagram.WinForm.Commands;

namespace Nevron.Examples.Diagram.WinForm
{
	/// <summary>
	/// Summary description for NDiagramExamplesConfig.
	/// </summary>
	public class NDiagramExamplesConfig : NExamplesConfig
	{
		#region Constructor

		public NDiagramExamplesConfig()
		{
		}


		#endregion

		#region Overrides

		public override void InitDefault()
		{
			base.InitDefault();

			m_SplashImage = NResourceHelper.BitmapFromResource(typeof(NMainForm), "DiagramSplash.png", "Nevron.Examples.Diagram.WinForm.Resources");

			m_sExamplesNamespace = "Nevron.Examples.Diagram.WinForm";
			m_sTreeResource = "ExamplesTree.xml";
			m_sTreeResourcePath = "Nevron.Examples.Diagram.WinForm.Resources";
			m_sFormText = "Nevron Diagram for Windows Forms - Part of Nevron .NET Vision - Examples";

			m_FormIcon = NResourceHelper.IconFromResource(typeof(NMainForm), "Diagram.ico", "Nevron.Examples.Diagram.WinForm.Resources");

			m_sFeedbackString = "mailto:support@nevron.com?subject=Diagram Feedback";

			m_iExampleTreeNodeImageIndex = 11;
			m_iExampleTreeNodeSelectedImageIndex = 12;
		
			m_EmbeddedResourcesAssembly = this.GetType().Assembly;
			m_LayoutStrategy = new NDiagramExamplesLayoutStrategy();

			ProductLogo = NResourceHelper.BitmapFromResource(typeof(NMainForm), "DiagramLogo.png", "Nevron.Examples.Diagram.WinForm.Resources");
			ProductName = "Nevron Diagram for .NET";
			ProductAssemblies = new System.Reflection.Assembly[]{typeof(NDiagramCommandBarsManager).Assembly};
		}


		#endregion
	}
}